packages/office-ui-fabric-react/src/components/Dialog/Dialog.base.tsx

```diff
@@ -108,8 +129,8 @@ export class DialogBase extends BaseComponent<IDialogProps, {}> {
         isDarkOverlay={ isDarkOverlay !== undefined ? isDarkOverlay : modalProps!.isDarkOverlay }
         isBlocking={ isBlocking !== undefined ? isBlocking : modalProps!.isBlocking }
         isOpen={ isOpen !== undefined ? isOpen : !hidden }
-        className={ css('ms-Dialog', className ? className : modalProps!.className) }
```

Comment:
Do we want to keep these className fallbacks?

Reply:
Yeah, I'll make a new fallback for all the classNames.